specs/exec-engine.md

```diff
@@ -196,18 +198,19 @@ Within the rollup, the types of forkchoice updates translate as:
 - `finalizedBlockHash`: irreversible block hash, matches lower boundary of the dispute period.
 
 To support rollup functionality, one backwards-compatible change is introduced
-to [`engine_forkchoiceUpdatedV2`][engine_forkchoiceUpdatedV2]: the extended `PayloadAttributesV2`
+to [`engine_forkchoiceUpdatedV3`][engine_forkchoiceUpdatedV3]: the extended `PayloadAttributesV3`
 
-#### Extended PayloadAttributesV2
+#### Extended PayloadAttributesV3
 
-[`PayloadAttributesV2`][PayloadAttributesV2] is extended to:
+[`PayloadAttributesV3`][PayloadAttributesV3] is extended to:
 
 ```js
-PayloadAttributesV2: {
+PayloadAttributesV3: {
```

Comment:
v2 is still required for bedrock and canyon?

Reply:
You can use the V3 payload, you just have to omit the parentBeaconBlockRoot. Similar to when we went from Bedrock to Canyon, you could use V2 for Bedrock, but had to make sure withdrawals == nil.